lib/cli-entry.js

```diff
@@ -1,6 +1,3 @@
-/* eslint-disable max-len */
```

Comment:
can we, like, just update our main linting rules now?

Reply:
We can set a different value for comments

https://eslint.org/docs/latest/rules/max-len

Reply:
https://github.com/npm/eslint-config/pull/102